recipes/poco/all/test_package/CMakeLists.txt

```diff
@@ -46,10 +46,14 @@ option(TEST_UTIL "Test Util")
 if(TEST_UTIL)
     add_executable(util test_util.cpp)
     target_link_libraries(util ${CONAN_LIBS})
+    if(MINGW)
+        target_link_options(util PRIVATE -municode)
+    endif()
 endif()
 
 option(TEST_SQLITE "Test Sqlite")
 if(TEST_SQLITE)
+    cmake_minimum_required(VERSION 3.13)
```

Comment:
Why 3.13? Why hooks didn't report an error for `cmake_minimum_required(VERSION 3.0)`?

This CMakeLists could be simplified, specially the top part:

```cmake
cmake_minimum_required(VERSION 3.0)
project(MyHello)
macro(use_cxx11)
  if(CMAKE_VERSION VERSION_LESS "3.1")
    if(CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
      set (CMAKE_CXX_FLAGS "--std=gnu++11 ${CMAKE_CXX_FLAGS}")
    endif()
  else()
    set(CMAKE_CXX_STANDARD 11)
  endif()
endmacro()
use_cxx11()
```

Reply:
I don't remember this, it looks unnecessary indeed.
I think it can be removed + the macro at the top can just be replaced with `set(CMAKE_CXX_STANDARD 11)` when bumping the minimum cmake version to 3.1.